docs/source/how-to/deploying-substra/upgrade-notes.rst

```diff
@@ -4,6 +4,102 @@
+
+Otherwise, you must update the data format. There are many ways to do this; here we give you a simple way which is acceptable if:
+
+  * Have small amounts of data
+  * Have access to the clusters
+  * Downtime is acceptable
+
+It must be repeated on each Substra node and orchestrator.
+
+  .. warning::
+
+    Before going further, make sure no operations are in progress and cordon the node.
+
+#. Gather required info:
+
+   * database credentials (here ``$PG_USER`` and an associated password)
+   * Kubernetes namespace (here ``$NAMESPACE``)
+   * Helm release (hereafter ``$RELEASE_NAME``, can be obtained from ``helm get release -A``)
+   * hostname of the Postgres service (hereafter ``$HOST``, can be obtained from ``kubectl get svc -n $NAMESPACE``)
+   * name of the database in Postgres: by default ``orchestrator`` for the orchestrator, and ``substra`` for the backend (hereafter ``$DB``)
+
+#. Deploy a psql client (we will call it ``postgres-backup``) in the namespace
+
+   .. code-block:: bash
+
+      kubectl apply -f -n $NAMESPACE - << 'EOF'
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        name: postgres-backup
+      spec:
+        containers:
+          - name: postgres
+            image: postgres
+            command: ["sleep", "infinity"]
+      EOF
+
+#. Launch a shell from **within the postgres-backup pod** and dump the DB:
+
+   .. code-block:: bash
+
+      pg_dump $DB --host=$HOST -U $PG_USER -f /dump.sql --clean --create
+
+#. (OPTIONAL) Retrieve the dump:
+
+   .. code-block:: bash
+
+      kubectl --retries 10 cp $NAMESPACE/postgres-backup:/dump.sql dump.sql
+      sha1sum dump.sql # check it matches with the source (use `shasum` on Mac OS)
+
+#. Delete the Postgres StatefulSet and PVC
+
+   This depends on your particular set-up but it should look like this:
+
+   .. code-block:: bash
+
+      kubectl delete -n substra statefulset $RELEASE_NAME-postgresql
+      kubectl delete -n substra pvc data-$RELEASE_NAME-postgresql-0
+
+#. Perform the database upgrade
+
+   Note versions ``8.0.0`` and ``23.0.0`` are used: they contain the database upgrade. The apps upgrade are done in ``8.0.1`` and ``23.0.2``.
+
+   You can get values with ``helm get values``
+
+   Orchestrator:
+
+   .. code-block:: bash
+
+      helm upgrade -n $NAMESPACE $RELEASE_NAME https://github.com/Substra/charts/raw/main/orchestrator-8.0.0.tgz --values orc-values.yaml
+
+   Backend:
+
+   .. code-block:: bash
+
+      helm upgrade -n $NAMESPACE $RELEASE_NAME https://github.com/Substra/charts/raw/main/substra-backend-23.0.0.tgz --values backend-values.yaml
+
+#. Delete the applicative ``deployments`` and ``statefulset`` to avoid them polluting the database (``orchestrator-server``, ``backend-server``, ``backend-worker``, ...)
+
+#. Launch a shell from **within the postgres-backup pod** and load the dump:
+
+
+   .. code-block:: bash
+
+      psql --host=$HOST -U $DB_USER < /dump.sql
+
+#. Perform final upgrade as normal
```

Comment:
I feel like the [important information](https://github.com/Substra/substra-documentation/blob/chore/release-0.34.0/docs/source/how-to/deploying-substra/upgrade-notes.rst?plain=1#L22) displayed before the warning could belong here instead: 
```suggestion
#. Perform final upgrade as normal

#. Repeat on each Susbtra node, including the orchestrator (if standalone)
```

---

Else, it might need a second warning, eg something like:

```rst
  .. warning:: 
  
      It must be repeated on each Substra node and orchestrator.
```